pure.zsh

```diff
@@ -270,7 +270,7 @@ prompt_pure_async_vcs_info() {
 	# Export branch (%b), Git toplevel (%R), action (rebase/cherry-pick) (%a),
 	# and stash information via misc (%m).
 	zstyle ':vcs_info:git*' formats '%b' '%R' '%a' '%m'
-	zstyle ':vcs_info:git*' actionformats '%b' '%R' '%a' '%m'
+	zstyle ':vcs_info:git*' actionformats '%b' '%R' '%a'
```

Comment:
We should probably only modify the formats (and `max-exports`) when stash is actually enabled.